src/styles/index.ts

```diff
@@ -3712,7 +3712,16 @@ const styles = (theme: ThemeColors) =>
             paddingTop: variables.searchListContentMarginTop,
         },
 
-        searchTopBarStyle: {
+        narrowSearchHeaderStyle: {
+            paddingTop: 1,
+            flex: 1,
+        },
+
+        typePopoverButtonStyle: {
```

Comment:
I know that we have a preference to use style props/consts but I think if it's just a class for 1 line of styles, then maybe it's better to just inline `theme.sidebarHover` in the component that it's used?